changelog/content/released/v2.15.0-scraper.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Please update https://github.com/tomkerkhove/promitor/blob/master/changelog/content/experimental/unreleased.md instead, this is not shipped yet